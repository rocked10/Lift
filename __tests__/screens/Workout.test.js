import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Workout from '../../screens/workout';

describe('Testing Workout Screen', () => {
    const wrapper = renderer.create(<Workout />);

    it('Should render', () => {
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});