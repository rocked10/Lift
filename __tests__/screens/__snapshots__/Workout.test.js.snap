// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Workout Screen Should render 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 16,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "flex-end",
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
  <RCTScrollView
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsVerticalScrollIndicator={false}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
